scripts/compile_tests/common.py

```diff
@@ -43,10 +43,7 @@ def find(testcase, condition):
 
 def skipped_test(testcase):
     def condition(children):
-        tags = [child.tag for child in children]
-        if "skipped" in tags:
-            return True
-        return False
+        return "skipped" in {child.tag for child in children}
```

Comment:
Replacing list with set is a bit unrelated, though reasonable
```suggestion
        return "skipped" in {child.tag for child in children}
```